app/eventyay/control/forms/event.py

```diff
@@ -99,6 +110,26 @@ def __init__(self, *args, **kwargs):
             self.fields['organizer'].initial = qs.first()
             self.fields['organizer'].required = False
 
+    def clean(self):
+        cleaned_data = super().clean()
+        locales = cleaned_data.get('locales', [])
+        content_locales = cleaned_data.get('content_locales')
+        
+        if not isinstance(locales, list):
```

Comment:
The `MultipleChoiceField` already guarantees this value to be a list, the `isinstance` is not necessary.
If your purpose is to help Pyright / your IDE infer type correctly, you can use:

```py
assert isinstance(locales, list)
```

to keep the code flat.